tc/sgx/trusted_worker_manager/enclave/work_order_data_handler.h

```diff
@@ -51,8 +51,8 @@ namespace tcf {
 
             void ComputeOutputHash();
             void VerifyInputHash(ByteArray input_data, ByteArray input_hash);
-            void DecryptInputData(ByteArray encrypted_input_data, std::string iv_str);
-            std::string EncryptOutputData(std::string iv_str);
+            void DecryptInputData(ByteArray decryption_key, std::string iv, ByteArray encrypted_input_data);
```

Comment:
Make key and iv is member variables and use it instead of passing it as argument to this function. Write setters for setting key and iv.